### ISSUE TITLE
CheckBox works only if using iconFamily 'Ionicons'

### ISSUE THREAD

**Issue by yossijacob** [GeekyAnts/NativeBase#531]
native-base@2.0.7
react@15.4.2
react-native@0.41.1
testing on android 6

When i change the iconFamily in the material.js from 'Ionicons' to 'MaterialIcons' 

`iconFamily: 'MaterialIcons'`

Then the checkbox icon is not working anymore.
I use this theme with the StyleProvider

```
import theme from '../native-base-theme/variables/material';
<StyleProvider style={getTheme(theme)}>
...
```

As i can see here https://github.com/GeekyAnts/NativeBase/blob/master/src/basic/Checkbox.js#L46
It will only work if the icon set has icon with the name 'md-checkmark'
`<IconNB name={((platform === 'ios') && (platformStyle !== 'material')) ? 'ios-checkmark-outline' : 'md-checkmark'} />
`
In earlier versions i have used the 'MaterialIcons' and it worked fine.

Thanks !

**Comment by sankhadeeproy007** [comment]
Will look into this. Thanks for reporting!

**Comment by sankhadeeproy007** [comment]
Fixed with 2.0.8
